tests/default/_core/search/aggregations/bucket.yaml

```
@@ -0,0 +1,49 @@
+$schema: ../../../../../json_schemas/test_story.schema.yaml
+
+description: Test aggregation within bucket aggregation.
+warnings:
+  invalid-path-detected: false
+prologues:
+  - path: /_bulk
+    method: POST
+    parameters:
+      refresh: true
+    request:
+      content_type: application/x-ndjson
+      payload:
+        - {create: {_index: movies}}
+        - {title: Pearl Harbour, directorId: 445212, cost: 2000, isCostHidden: true}
```

Is this closer to convention to other places? (check)

```suggestion
        - {title: Pearl Harbour, director_id: 445212, cost: 2000, cost_hidden: true}
```